docs/kit/logistics/kit-testing.md

```diff
@@ -0,0 +1,221 @@
+- A USB stick
+
+```python
+from pathlib import Path
+import RPi.GPIO as GPIO
+asset_file = Path("/proc/device-tree/hat/custom_0")
+if asset_file.exists():
+    asset = asset_file.read_text()
+    # append code to seen file
+    with open("tested_KCHs.txt", "a") as f:
+        f.write(asset)
+        f.write("\n")
+
+    GPIO.setmode(GPIO.BCM)
+    GPIO.setup([10, 23, 18], GPIO.OUT, initial=GPIO.HIGH)
+else:
+    raise RuntimeError("No KCH found")
+```
+
+Test steps:
+
+1. Copy the above block of code into a file called `robot.py`. This should be the only file on the USB stick.
+2. Check that the Pi has an SD card inserted. Otherwise use an SD card from another Pi.
+3. Plug the USB stick into the Pi.
+4. Connect the 12V power supply to the brain board's Camcon connector.
+5. 3 Green LEDs on the Camcon end of the board will light.
+    - If they don't light immediately this board has failed, go to step 8.
+6. The 5 boot LEDs will light in order.
+    - If the first LED has not lit within 20 seconds this board has failed, go to step 8.
+    - If all 5 LEDs have not lit within approximately a minute this board has failed, go to step 8.
+7. The OK LED should then light a few colours before stabilising on green.
+    - If the LED stabilises on red this board has failed, go to step 8.
+    - If the LED has not lit within 20 seconds try replugging the USB stick. If it continues to not light this board has failed, go to step 8.
+8. Remove board making a separate pile for boards that failed.
+9. Repeat steps 2-8 for the next board.
+10. Copy the `tested_KCHs.txt` file off the USB stick. This is the KCH asset codes of all the passing brain boards.
+
+### Power Board
+
+Here we are testing all outputs can be enabled and current sense is functioning.
+Additionally, we test the buzzer, LEDs and start button are functioning and finally we test both the software and hardware undervoltage protection.
+
+For this test you will need:
+
+- The power board testing jig, pictured below
+- High current power supply (Dell DPS-750TB)
+- USB controllable power supply (Tenma 72-2545)
+- Power supply switcher
+
+![Power board jig](./img/power_jig.jpeg)
+<!-- TODO add picture of switcher connection -->
+
+!!! note
+
+    While testing undervoltage, errors will be printed about failing to connect to the board. This is expected behaviour.
+
+Test steps:
+
+1. Connect up the high current PSU and USB controlled PSU to the switcher circuit.
+2. Connect the switcher circuit to the test jig.
+3. Set the USB controlled PSU to 12V and 3A.
+4. Set all switches on the jig to up position.
+5. Set the on/off switch to on.
+6. Connect a USB cable to the board to test.
+7. Slide board into the jig from the top.
+8. Insert both sets of side plugs, ensuring they are fully inserted.
+9. Connect the board's XT60 connecter to the rear of the jig.
+10. Run the script and follow its instructions:<br>
+    `kit_test power_v4 --fw-ver 4.4.2 --test-uvlo --log power_testing.csv`
```

Comment:
Question: I assume from context "uvlo" is Under-Voltage LockOut, and is a fairly standard acronym?

Reply:
Yes